src/magic_folder/magicfolderdb.py

```diff
@@ -202,3 +216,39 @@ def did_upload_version(self, relpath_u, version, last_uploaded_uri, last_downloa
                                      relpath_u))
                 action.add_success_fields(insert_or_update=u"update")
             self.connection.commit()
+
+    def store_local_snapshot(self, serialized_snapshot, path, foldername):
+        """
```

Comment:
the args should be documented .. and is "path" a mangled-path or a full absolute path or ..? If it's a filesystem path it sholud maybe be a `FilePath` instance. If it's the mangled path, maybe use `name` (or `snapshot_name`) so we're more-consistent with `LocalSnapshot.name`?

Reply:
That seem like a good idea. It was simple enough that, I went ahead and implemented that in this PR.

Reply:
On second thought, I thought I would merge in the changes and submit those changes as a separate PR.